lightly/cli/upload_cli.py

```diff
@@ -10,20 +10,35 @@
 import csv
 import json
 import os
+import warnings
 from datetime import datetime
+from typing import Union
 
 import hydra
 
 import torchvision
 from torch.utils.hipify.hipify_python import bcolors
 
+from lightly.api.api_workflow_upload_embeddings import \
+    EmbeddingDoesNotExistError
 from lightly.cli._helpers import fix_input_path, print_as_warning, cpu_count
 
 from lightly.api.api_workflow_client import ApiWorkflowClient
 from lightly.data import LightlyDataset
+SUCCESS_RETURN_VALUE = "Success"
```

Comment:
This is only used for testing right?

Reply:
Yes, it is needed to ensure that the "working case" tests actually work and do not exit earlier with warning.